### PR TITLE
78581 scenario verification functional tests

### PR DESCRIPTION
**Pull request by Sadha1501493**
Branch has logic to verify XMLs of below 2 scenarios:
1. New cell Scenario
2. Cancel and replaced Scenario

### CODE REVIEW

**Review by RaviK2233 — APPROVED** [review]
Reviewed FT and looks good

**Review by RaviK2233 — COMMENTED** [review]
Hi @Sadha1501493 , some of the checks have failed. Could you please resolve them.

File: tests/UKHO.ERPFacade.API.FunctionalTests/Model/JsonPayloadHelper.cs [hunk 1]
```diff
@@ -0,0 +1,277 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UKHO.ERPFacade.API.FunctionalTests.Model
+{
+    public class JsonPayloadHelper
```
[the remark this answers is not in the document]
Comment (Sadha1501493) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Model/JsonPayloadHelper.cs:10]: This class is created for .json file deserialization. So, from maintenance perspective good to have in one file.
But still let me know if split is needed.

**Review by jrippington — CHANGES_REQUESTED** [review]
Thanks, please can you resolve the merge conflicts?

**Review by jrippington — APPROVED** [review]

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**